components/prospects/ProfileActions.tsx

```diff
@@ -0,0 +1,81 @@
+import { grey } from '@mui/material/colors';
+import ConfirmModal from '../helper/ConfirmModal';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import { useState } from 'react'
+
+function ProfileActions({ id, name, options }) {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [modalOpen, setModalOpen] = useState(false);
+  const [selectedOption, setSelectedOption] = useState
+  <{ label: string; action: () => void }>(null);
```

Comment:
Indentation?! Should be with a tab to the right and `useState` should also be on the next line